src/main/java/com/exactpro/th2/common/event/Event.java

```diff
@@ -194,6 +196,19 @@ public Event bodyData(IBodyData bodyData) {
         return this;
     }
 
+    /**
+     * Adds exception with all causes to the body data as series of messages
+     * @return current event
+     */
+    public Event exception(@NotNull Throwable throwable) {
```

Comment:
How about adding another method `exceptionWithStacktrace` or add the parameter to this method that switches how the exception will look in the report and allows the user to add a full stack trace instead of exception messages only?